dist/index.html

```diff
@@ -143,12 +144,13 @@ <h1 id="sponsors" class="sponsors-title">Last edition's sponsors</h1>
                 <a href="http://everis.com/" target="_blank"><img class="ml mr" src="assets/img/EVERIS.png"></a>
                 <div class="separator"></div>
             </div>
+            -->
             <div class="kilo col-md-12">
-                <a href="https://scalingo.com/" target="_blank"><img class="ml mr" src="assets/img/scalingo.png"></a>
-                <a href="https://www.signaturit.com/" target="_blank"><img class="ml mr" src="assets/img/signaturit.png"></a>
+                <a href="https://www.doist.com/" target="_blank"><img class="ml mr" src="assets/img/doist.png"></a>
             </div>
+            <!--
             <div class="kilo col-md-12">
-                <a href="https://www.bloomberg.com/careers/technology/engineering" target="_blank"><img class="ml mr" src="assets/img/BLOOMBERG.png"></a>
+                <a href="https://www.bloomberg.com/careers/technology/engineering" target="_blank"><img class="ml mr" src="assets/img/Doist.png"></a>
```

Comment:
True. Link has to be changed. And yeah, past sponsors should be removed. I guess @casassg just wanted to keep the structure for later use. We could just leave the `div`s of the different levels of sponsors commented